apps/framework-docs/src/pages/building.mdx

```diff
@@ -0,0 +1,29 @@
+import { Cards, Card } from 'nextra/components'
+
+# Building a Data Service
+MooseJS helps you build data intensive applications and service. In its current state, MooseJS can't be used to build a full fledged web application. However, it can be used to build a data service which you can use to power adhoc analytics use cases.
+
+## What is a data service?
+With the MooseJS's current functionality a data service can be thought of as a microservice that's used to capture, store and serve data. It interfaces with best in-class data infrastructure and allows you to build a data warehouse wihtout any data engineering skills. A service is composed of:
+
+- A series of data models which define the structure of the data you want to caputre
+- An ingestion API to capture data from other applications or services
+- A storage layer that queues up data on ingestion and stores it for later processing
+
+We recommend that you use all the standard software development practices to build your service. 
```

Comment:
Hum - reading this as a software engineer, I could be offended - of course I want to use best practices. I would remove this sentence.

Instead, I would flip it and say something along the lines of:

`Thanks to 'moose', you get to use the same best practices you use for your application layer in your analytics and data warehousing layer.`